packages/dashboard/src/components/sidePanel/index.tsx

```diff
@@ -28,6 +30,7 @@ const SidePanel: FC<{ messageOverrides: DashboardMessages }> = ({ messageOverrid
         <BaseSettings messageOverrides={messageOverrides} />
         {isTextWidget && <TextSettings messageOverride={messageOverrides} />}
         {isTextWidget && <LinkSettings messageOverride={messageOverrides} />}
+        {isInputWidget && <InputSettings messageOverride={messageOverrides} />}
```

Comment:
Agreed. I like this idea, it is more scalable than what we have been doing so far in the sidePanel. I would say it is out of scope for this PR but we should keep track of this and tackle it at some point.

Reply:
We will tackle the way we render this component when we define the customization api. It will look more like your suggestion @tjuranek than a bunch of if statements. We are still trying to figure out the best way to write that though